sorting-algorithms/problem-set-sorting-algorithms.md

```diff
@@ -272,7 +67,7 @@ For algorithms with equal time complexity, order them in alphabetical order.
 
 There's nothing like getting familiar with sorting algorithms than to implement it yourself!
```

Comment:
Outside your change, but this jumped out me as being really strangely worded. Could we slip this in?

```suggestion
There's no better way to get familiar with sorting algorithms than to implement one ourselves!
```